src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -499,7 +531,7 @@ def extract_return_input_types(
                 True,
                 False,
                 res_return_type.element_type,
-                [tp.element_type for tp in res_input_types],
+                [tp.element_type for tp in res_input_types],  # type: ignore
```

Comment:
All the elements in this list have been validated to be `PandasSeriesType`, but mypy isn't picking up on that.